introduction.md

```diff
@@ -0,0 +1 @@
+# Helo, my name is Vina and I'm doing Code School Mastering Github course from a nice weather of Singapore. Yoroshiku!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.